nodejs-react-bootstrap/nodejs-react-bootstrap.spec

```diff
@@ -124,4 +122,3 @@ rm -rf %{buildroot} %{npm_cache_dir}
 
 * Fri Oct 14 2016 Dominic Cleal <dominic@cleal.org> 0.30.5-1
 - new package built with tito
-
```

Comment:
Should we fix this in the template/update script so we don't go back and forth all the time?

Reply:
Definitely, I removed this by hand, it isn't very relevant though

Reply:
tito adds it again so you'll just end up with noise in the git log.